src/main/java/teammates/ui/controller/InstructorHomeCourseAjaxPageData.java

```diff
@@ -146,7 +146,7 @@ private void addAttributeIf(boolean shouldAdd, ElementTag button, String key, St
                     getInstructorFeedbackStatsLink(session.courseId, session.feedbackSessionName),
                     isRecent,
                     getInstructorFeedbackSessionActions(
-                            session, false, instructor));
+                            session, true, instructor));
```

Comment:
oh. this is to set the isHome to true so the correct return url can be generated at the InstructorFeedbackSessionActions.java